src/bot/constants.py

```diff
@@ -33,6 +33,18 @@ class _Miscellaneous(EnvConfig):
 
 Miscellaneous = _Miscellaneous()
 
+
+class _ThreatIntelFeed(EnvConfig, env_prefix="tif_"):
+    """Threat Intelligence Feed Configuration."""
+
+    repository: str = "pypi/pypi-observation-reports-private"
+    interval: int = 10 * 60  # 10 minutes
```

Comment:
```suggestion
    interval: int = 60 * 60  # 1 hour
```

We don't need this to be updated that frequently-- they're added in bulk.